filesystem/build.gradle

```diff
@@ -14,7 +14,7 @@ dependencies {
     implementation libraries.okio
     implementation libraries.coroutinesCore
     implementation libraries.cache
-    implementation project(path: ':store')
+    implementation project(path: ':store4')
```

Comment:
change back, we keep module as store but artifacts as store5, same for the package

Reply:
Hey I don't follow - My understanding was current `:store` becomes `:store4`. Then Store 5 becomes new `:store`. And `:filesystem`, `:store-rx2`, and `:store-rx3` depend on `:store4` rather than `:store`. What am I missing?

Reply:
https://github.com/MobileNativeFoundation/Store/tree/main/store/src/main/java/com/dropbox/android/external/store4 rename this package to store5. Rename the maven artifacts to store5. We don't need to have store4 and store5 in repo at same time. Your change can be a breaking change. The important thing is someone can add the store4 and store5 artifacts at same time